### PR TITLE
Add comment

### PR DESCRIPTION
**Pull request by jahjam**

### CODE REVIEW

**Comment by NRMSMITH** [comment]
- [ ] Please have a message such as 'comment posted' appear, so that a screen reader will pick up on it - it won't necessarily re-read the rendered comments.
- [ ] When I make a comment while offline (or while the server is offline), I should be able to see it appear on the comment list, and then disappear off and a message saying 'something went wrong' (or something like that) should appear.

Please resubmit after these are complete, also I love the logo design it's very nice 🚀

File: src/feature/Comments/Comments.tsx [hunk 1]
```diff
@@ -25,12 +36,61 @@ const Comments = ({ reviewId }: Props) => {
...
+
+            {isError && <span>{errorMsg}</span>}
+
+            {commentPosted && <p>Comment Posted!</p>}
+
+            {isLoading ? (
+              <Button type="button">Posting...</Button>
+            ) : (
+              <Button type="submit">Add Comment</Button>
+            )}
```
Comment (popatre) [inline src/feature/Comments/Comments.tsx:81]: This is nice!

File: src/feature/Comments/Comments.tsx [hunk 2]
```diff
@@ -25,12 +36,61 @@ const Comments = ({ reviewId }: Props) => {
...
+  };
+
   return (
     <Styled.Comments direction="column">
+      {isLoggedIn ? (
+        <>
+          <h3>Add A Comment</h3>
+          <Styled.Form onSubmit={handleSubmit} direction="column" gap={0.8}>
+            <label htmlFor="body">Comment:</label>
+            <textarea ref={body} id="body"></textarea>
```
Comment (popatre) [inline src/feature/Comments/Comments.tsx:71]: Nice use of the ref hook!

**Review by popatre — DISMISSED** [review]
Looks good. Some nice code there!

it a small thing, but the error message when offline is confusing for an end user. It would be better to just say `comments cannot be empty` or something like that

**Pull request merged**